### PR TITLE
Merge fenix quarantine

### PR DESCRIPTION
**Pull request by Pike**

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -539,7 +549,101 @@
...
+
+    <!-- Title for the Delete browsing data preference -->
+    <string name="preferences_delete_browsing_data">Delete browsing data</string>
+    <!-- Title for the tabs item in Delete browsing data -->
+    <string name="preferences_delete_browsing_data_tabs_title">Open Tabs</string>
+    <!-- Subtitle for the tabs item in Delete browsing data, parameter will be replaced with the number of open tabs -->
+    <string name="preferences_delete_browsing_data_tabs_subtitle">%d tabs</string>
+    <!-- Title for the data and history items in Delete browsing data -->
+    <string name="preferences_delete_browsing_data_browsing_data_title">Browsing history and site data</string>
+    <!-- Subtitle for the data and history items item in Delete browsing data, parameter will be replaced with the
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: Nit: "(...)items item(...)"

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -147,8 +151,10 @@
     <string name="preferences_sync_bookmarks">Bookmarks</string>
     <!-- Preference for signing out -->
     <string name="preferences_sign_out">Sign out</string>
+    <!-- Preference displays and allows changing current FxA device name -->
+    <string name="preferences_sync_device_name">Device name</string>
     <!-- Label indicating that sync is in progress -->
-    <string name="sync_syncing">Syncing</string>
+    <string name="sync_syncing">Syncing…</string>
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: This will need a string ID change so that localizers see they need to update this string

**Comment by Pike** [comment]
Latest update is https://github.com/mozilla-l10n/android-l10n/compare/4b58bfd..7388bf8, didn't spot new issues.

Changes requested are worked on in https://github.com/mozilla-mobile/fenix/pull/2950.

**Comment by Delphine** [comment]
This lgtm, thanks

**Pull request merged**